src/coreclr/jit/dataflow.h

```diff
@@ -58,19 +58,47 @@ void DataFlow::ForwardAnalysis(TCallback& callback)
         }
         else
         {
-            FlowEdge* preds = m_pCompiler->BlockPredsWithEH(block);
-            for (FlowEdge* pred = preds; pred; pred = pred->getNextPredEdge())
+            for (FlowEdge* pred = block->bbPreds; pred; pred = pred->getNextPredEdge())
```

Comment:
Nit: can this loop use the `PredEdges` iterator?

Reply:
Yeah, let me simplify this tomorrow.